templates/example_pixel_cell_clustering.ipynb

```diff
@@ -0,0 +1,679 @@
+{
```

Comment:
<p>We should replace this with a link to the documentation saying if people want to change default settings, they can. However, since these defaults rarely get changed, we should remove them from the notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>